src/app/badge/badge.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="body row">
+  <div class="col-sm-4">
+    <img [src]="data.image">
+  </div>
+  <div class="col-sm-8">
+    <h3>{{language?.headingLeft}}<span class="badge-name">{{data.name}}</span> {{language?.headingRight}}</h3>
+    <h3>{{language?.infokitMessage}}</h3>
+  </div>
+</div>
```

Comment:
new line

Reply:
@Raxerz I checked it and it has a new line. Please check again.

Reply:
Its not there @lunayach